hash.c

```diff
@@ -4158,11 +4167,35 @@ rb_hash_equal(VALUE hash1, VALUE hash2)
 
 /*
  *  call-seq:
- *     hash.eql?(other)  -> true or false
- *
- *  Returns <code>true</code> if <i>hash</i> and <i>other</i> are
- *  both hashes with the same content.
- *  The orders of each hashes are not compared.
+ *    hash.eql? other_hash -> true or false
```

Comment:
It doesn't have to be `other_hash`, it can be any object right?

Reply:
Fixed.